common/djangoapps/course_modes/models.py

```diff
@@ -54,7 +54,7 @@ class Meta(object):
         db_constraint=False,
         db_index=True,
         related_name='modes',
-        on_delete=models.CASCADE,
+        on_delete=models.DO_NOTHING,
```

Comment:
Is this change necessary now that we're not deleting?

Also, shouldn't this trigger a migration (even though it's a no-op in the DB)?

Reply:
Not necessary, but also more consistent w/ the other places that we FK onto CourseOverview. But yes, I should run `makemigrations`, good call.